config/routes.rb

```diff
@@ -86,7 +86,12 @@
   get '/login', to: 'sessions#new'
   get '/logout', to: 'sessions#destroy'
 
-  resources :csv_exports, only: [:index, :new, :create, :show]
+  resources :csv_exports, only: [:index, :new, :create, :show] do
```

Comment:
this looks unsymmetric ... can we re-use new and index with some kind of parameter ... otherwise create a new controller and make the existing one